datasets/lj_speech/README.md

```diff
@@ -18,6 +18,20 @@ source_datasets:
 task_categories:
 - automatic-speech-recognition
 task_ids: []
+train-eval-index:
+- config: main
+  task: automatic-speech-recognition
```

Comment:
The ASR support in AutoTrain is currently being reworked, so let's exclude speech datasets in the next batch (keep this one though!)